packages/medusa/src/api-v2/admin/price-lists/[id]/route.ts

```diff
@@ -29,3 +31,77 @@ export const GET = async (
 
   res.status(200).json({ price_list: priceList })
 }
+
+export const POST = async (
+  req: AuthenticatedMedusaRequest<AdminPostPriceListsPriceListReq>,
+  res: MedusaResponse
+) => {
+  const id = req.params.id
+  const workflow = updatePriceListsWorkflow(req.scope)
+
+  const [_, count] = await listPriceLists({
+    container: req.scope,
+    fields: ["id"],
+    variables: { filters: { id }, skip: 0, take: 1 },
+  })
+
+  if (count === 0) {
+    throw new MedusaError(
+      MedusaError.Types.NOT_FOUND,
+      `Price list with id: ${id} was not found`
+    )
+  }
```

Comment:
thought(non-blocking): Would it make sense to move these to the workflow level?

What happens if the price list does not exist in the workflow? Do we handle that gracefully?